## 91_build_pwdecrypt.patch by Paul Wise <pabs@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Enable building and installing pwdecrypt (Closes: #472303)

Index: nss/mozilla/security/nss/cmd/manifest.mn
===================================================================
--- nss.orig/mozilla/security/nss/cmd/manifest.mn	2011-11-05 17:03:08.430363145 +0100
+++ nss/mozilla/security/nss/cmd/manifest.mn	2011-11-05 17:03:26.446390163 +0100
@@ -66,6 +66,7 @@
  pk12util \
  pk11mode \
  pp  \
+ pwdecrypt \
  rsaperf \
  sdrtest \
  selfserv  \
